api/db/build.gradle

```diff
@@ -0,0 +1,21 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.liquibase:liquibase-gradle-plugin:1.2.4'
+        classpath 'mysql:mysql-connector-java:5.1.37'
+    }
+}
+apply plugin: 'org.liquibase.gradle'
+
+liquibase {
+    activities {
+        main {
+            changeLogFile 'changelog/db.changelog-master.xml'
+            url 'jdbc:mysql://localhost:3306/workbench'
```

Comment:
I mainly wanted to get something working for the M0 prototype so we can save cohorts and field sets.

Thinking it over, I think we may be better off having a Python or Java wrapper for this stuff, which would change it significantly. I'll give that a shot.